bin/perf_test.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import sys
+import time
+import random
+import string
+import socket
+from subprocess import call, check_output, STDOUT
+import xml.etree.ElementTree as ET
+
+# Carbon settings
+CARBON_SERVER = '0.0.0.0'
+CARBON_PORT = 2003
+
+sock = socket.socket()
+sock.connect((CARBON_SERVER, CARBON_PORT))
+
+BUCKET_NAMES = ['wednesday', 'gobig']
```

Comment:
I think we should know when testing how many objects are in a bucket.  Are we going to prefill these buckets with some number of objects?  Maybe we should name them with the target number of objects -- oneMillionObjectBucket, tenMillionObjectBucket?